kuma/static/styles/base/elements/forms.scss

```diff
@@ -88,6 +88,7 @@ input[type='button'] {
     }
 
     &.positive {
+        min-height: 33px;
```

Comment:
Nit: `33px` seems a little odd. I would suggest making it `32px`. Thanks @Mihir-Karbelkar

Reply:
Shall I ammend or git commit?